src/zeit/content/article/interfaces.py

```diff
@@ -239,9 +239,3 @@ class IBreakingNews(IArticle):
     def banner_matches(banner):
         """Returns True if the given banner content object refers to this
         breaking news article."""
-
-
-IBreakingNews.setTaggedValue(
-    'zeit.cms.addform', 'zeit.content.article.AddBreakingNews')
-IBreakingNews.setTaggedValue(
```

Comment:
Interessant, danke für die Klärung. :) Das heißt in diesem konkreten Fall, dass die beiden Aufrufe von `setTaggedValue` stehen bleiben sollten, richtig? Über die URLs werden jetzt halt echte Content-Typen angelegt.